TSEconomy/Lang/Lang_es.xml

```diff
@@ -0,0 +1,298 @@
+  <KeyValue>
+	  <!-- Description: Bank admin command alias-->
+    <Key>ba</Key>
+    <Value>ba</Value>
+  </KeyValue>
+  <KeyValue>
+	  <!-- Description: Bank admin subcommand-->
+    <Key>setbal</Key>
+    <Value>setbal</Value>
+  </KeyValue>
+  <KeyValue>
+	  <!-- Description: Bank admin subcommand-->
+    <Key>resetbal</Key>
+    <Value>resetbal</Value>
+  </KeyValue>
+  <KeyValue>
+	  <!-- Description: Bank admin subcommand-->
+    <Key>checkbal</Key>
+    <Value>checkbal</Value>
+  </KeyValue>
+  <KeyValue>
+	  <!-- Description: Error showed to the player when searching an inexistant player-->
+    <Key>[i:855]That player does not exist!</Key>
+    <Value>[i:855]¡Ese jugador no existe!</Value>
+  </KeyValue>
+  <KeyValue>
+	  <!-- Description: Error showed to the player when the currency parameter was not input-->
+    <Key>[i:855]Please enter a currency name!</Key>
+    <Value>[i:855]¡Ingrese un nombre para su moneda!</Value>
+  </KeyValue>
+	 <!-- Description: Showed to the player when the amount parameter was not set-->
+  <KeyValue>
+    <Key>[i:855]Please enter an amount to set!</Key>
+    <Value>[i:855]¡Ingrese una cantidad para establecer!</Value>
+  </KeyValue>
+  <KeyValue>
+	   <!-- Description: Logged message informing that an admin setted a user's balance
+	        {0} is the admin's name, {{0}} is the user's name, {{1}} is the old balance and {{2}} 
+			is the new one-->
+    <Key>The admin {0} set the user {{0}}'s balance from {{1}} to {{2}}.</Key>
+    <Value>El administrador {0} estableció el saldo del usuario {{0}} de {{1}} a {{2}}.</Value>
+  </KeyValue>
+  <KeyValue>
+	  <!-- Description: Showed to the player after an admin has their balance,
+	        {0} is the currency name, {1} is the new value and {2} is the admin's name-->
+    <Key>[i:855]Your balance for {0} was set to {1} by {2}</Key>
+    <Value>[i:855]Su saldo de {0} fue establecido en {1} por {2}</Value>
+  </KeyValue>
+  <KeyValue>
+	  <!-- Description: Showed to the player when they enter an invalid amount-->
+    <Key>[i:855]The amount you entered was invalid!</Key>
+    <Value>[i:855]¡La cantidad que ingresaste no es válida!</Value>
+  </KeyValue>
+  <KeyValue>
+	  <!-- Description: Logged message informing that an admin reset a user's balance,
+	        {0} is the admin's name, {{0}} is the user's name, {{1}} is the new balance
+			{{2}} is the old balance-->
+    <Key>The admin {0} reset the user {{0}}'s balance from {{2}} to {{1}}.</Key>
+    <Value>El administrador {0} restableció el saldo del usuario {{0}} de {{2}} a {{1}}.</Value>
+  </KeyValue>
+  <KeyValue>
+	  <!-- Description: message showed to the player after their balance has been reset
+	        {0} is the currency name, {1} is the admin's name-->
+    <Key>[i:855]Your balance for {0} was reset to 0 by {1}</Key>
+    <Value>[i:855]Su saldo de {0} fue restablecido a 0 por {1}</Value>
+  </KeyValue>
+  <KeyValue>
+	 <!-- Description: list currencies command name -->
+    <Key>listcurr</Key>
```

Comment:
maybe instead of storing command aliases within local file we can just make them configurable to fit the needs of languages?

Reply:
> hmmm, tho are the aliases hardcoded? like when the lang file is generated, the plugin will simply read to it so if the user wants to modify it they are free to do so, but i do see the use of putting them in the config file, afterall most users wont bother to check the localization file i assume

not technically, but the quantity is... and if the user only wants one, two, or maybe they want more that is currently impossible